templates/bundles/FOSUserBundle/ChangePassword/change_password.html.twig

```diff
@@ -0,0 +1,7 @@
+{% extends "@FOSUser/layout.html.twig" %}
+
+{% form_theme form 'bootstrap_3_layout.html.twig' %}
```

Comment:
I think this line should be moved to `change_password_content.html.twig`.
And then, `change_password.html.twig` doesn't need to be overriden 🙂